srf_generation/source_parameter_generation/uncertainties/versions/nhm_4_asperity.py

```diff
@@ -0,0 +1,71 @@
+"""A basic perturbator as an example and starting point"""
```

Comment:
I'm not sure this needs a completely separate file - just because we might end up with lots of these files for each feature. :thinking:

Reply:
Yeah, this was fairly lazy documentation. I'm dubious of putting the documentation too far away from the implementation though, to try and cut down on the amount of support calls. I agree having one for each feature would be too many though. Perhaps we can use the file as a showcase of all extra features to make it copy/paste able.

Reply:
Yeh, using this as a showcase is a good idea.